cairo/tests/ethereum/utils/test_numeric.py

```diff
@@ -0,0 +1,20 @@
+from hypothesis import given
+from hypothesis import strategies as st
+from starkware.cairo.lang.instances import PRIME
+
+from tests.utils.strategies import uint128
+
+
+class TestNumeric:
+    @given(a=uint128, b=uint128)
+    def test_min(self, cairo_run, a, b):
+        assert min(a, b) == cairo_run("test_min", a=a, b=b)
+
+    @given(
+        value=uint128,
+        div=st.integers(min_value=1, max_value=PRIME // (2**128 - 1) - 1),
+    )
+    def test_divmod(self, cairo_run, value, div):
```

Comment:
ik and have seen that the 0 case is handled correctly but should it be covered by a test

Reply:
`divmod(x, 0)` raises in python with `ZeroDivisionError` and in cairo with `AssertionError: div=0x0 is out of the valid range.`